### PR TITLE
Simple Dummy Run for mtbf operation

### PR DESCRIPTION
**Pull request by ypwalter**

### CODE REVIEW

**Comment by zapion** [comment]
First default.json should be renamed so it can be recognized as a testvars file.

File: mtbf_job_runner.py [hunk 1]
```diff
@@ -283,9 +283,11 @@ def check_version(self):
     def mtbf_options(self):
         ## load mtbf parameters
         if not 'MTBF_TIME' in os.environ:
-            logger.warning("MTBF_TIME is not set")
+            logger.info("MTBF_TIME is not set, using default value 120.")
+            os.environ["MTBF_TIME"] = "120"
```
Comment (zapion) [inline mtbf_job_runner.py]: Default value is 120 seconds it might not need to assign again.

File: mtbf_job_runner.py [hunk 2]
```diff
@@ -283,9 +283,11 @@ def check_version(self):
...
         if not 'MTBF_CONF' in os.environ:
-            logger.warning("MTBF_CONF is not set")
+            logger.info("MTBF_CONF is not set, using default.conf")
+            os.environ["MTBF_CONF"] = "default.conf"
```
Comment (zapion) [inline mtbf_job_runner.py]: Should it be a json file?

File: mtbf_job_runner.py [hunk 3]
```diff
@@ -294,12 +296,14 @@ def mtbf_options(self):
         mtbf_testvars_dir = "/mnt/mtbf_shared/testvars"
         if not hasattr(self.options, 'testvars') or not self.options.testvars:
             testvars = os.path.join(mtbf_testvars_dir, "testvars_" + self.serial + ".json")
-            logger.info("testvar is [" + testvars + "]")
             if os.path.exists(testvars):
                 self.options.testvars = parser.testvars = testvars
                 logger.info("testvar [" + testvars + "] found")
             else:
-                raise AttributeError("testvars[" + testvars + "] doesn't exist")
+                if os.path.exists(os.path.join(os.getcwd(), "default.json")):
```
Comment (zapion) [inline mtbf_job_runner.py]: Recommend to use elif and avoid more nested structure.

File: mtbf_job_runner.py [hunk 4]
```diff
@@ -380,4 +384,13 @@ def run(self):
 
 if __name__ == '__main__':
     mjr = MtbfJobRunner()
+
+    ## use default settings if no assigned
+    settings_argv = False
```
Comment (zapion) [inline mtbf_job_runner.py]: Highly recommend not to modify sys argv here since we have full control of the stack.

**Comment by ypwalter** [comment]
too out-of-date to be merged.

**Pull request closed without merging**